src/client/mod.rs

```diff
@@ -127,6 +130,19 @@ impl<T: Transport, L: LoginCredentials> TwitchIRCClient<T, L> {
         self.privmsg(channel_login, format!(". {}", message)).await
     }
 
+    /// Replies to a given message in Twitch chat.
+    ///
+    /// Sends a message in the channel of the given message, tagging the original message and it's sender.
+    pub async fn reply(&self, privmsg: PrivmsgMessage, message: String) -> Result<(), Error<T, L>> {
+        let irc_message = IRCMessage::new(
+            IRCTags::parse(&format!("reply-parent-msg-id={}", privmsg.message_id)),
```

Comment:
I don't like this at all - parsing a formatted string. You can create new IRCTags and then add a single entry to them as needed.

Reply:
Done.